db-pull-on-boot/templates/db-pull.sh.j2

```diff
@@ -16,6 +16,11 @@ trap cleanup EXIT
 
 touch $LOCK_FILE
 
+echo "Downloading s3 database..."
+aws s3 cp "s3://{{ blockchain_backup_bucket }}/{{ network }}/blockchain_{{ db_version }}_{{ network }}.sqlite" "{{ chia_root }}/blockchain_{{ db_version }}_{{ network }}.sqlite"
```

Comment:
Make sure to also pull the other two files https://github.com/Chia-Network/ansible-roles/blob/main/chia-blockchain/tasks/main.yml#L99